index.html

```diff
@@ -1830,9 +1830,60 @@
       <section>
         <h3>Bearer Claims</h3>
 
-        <p class="issue">
-Bearer claims containing PII or unique identifiers can be correlated.
-Bearer claims can be tracked based on usage patterns.
+        <p>
+A <em>bearer claim</em> is a privacy enhancing piece of information,
+such as a concert ticket, that entitles the <a>holder</a> of the claim to a
+particular resource without divulging sensitive information about the holder.
+        </p>
+
+        <p>
+Verifiable Claims that are bearer claims are possible by not specifying the
+<a>subject</a> identifier in the <code>claim</code> property. For example, the
+following Verifiable Credential is a bearer claim:
+        </p>
+
+        <pre class="example nohighlight" title="Usage of issuer properties">
+{
+  "id": "http://dmv.example.gov/credentials/temporary/28934792387492384",
+  "type": ["Credential", "ProofOfAgeCredential"],
+  "issuer": "https://dmv.example.gov/issuers/14",
+  "issued": "2017-10-22T12:23:48Z",
+  "claim": {
+    <span class="comment">// note that the 'id' property is not specified for bearer claims</span>
+    "ageOver": 21
+  },
+  "signature": { ... }
+}
+        </pre>
+
+        <p>
+While bearer claims can be privacy enhancing, their use must be carefully
+crafted to not accidentally divulge more information than the holder of the
+claim expects. For example, repeated use of the same bearer claim across
+sites enables each site to potentially collude to track or correlate the holder.
+Additionally, information that may seem non-identifying such as a birth date
+and zip code can be used to statistically identify an individual when used
+together in the same claim or session.
+        </p>
+
+        <p>
+<a>Issuers</a> of bearer claims SHOULD ensure that bearer claims that are
+expected to provide privacy enhancing benefits 1) are single use, when possible,
+2) do not contain personally identifying information, and 3) are not
+correlatable.
+        </p>
+
+        <p>
+<a>Holders</a> SHOULD be warned by their software if bearer claims containing
+sensitive information are issued or requested, or if there is a correlation
+risk when combining two separate bearer claims across one or more sessions.
+While it may be impossible to detect all correlation risks, there are a few
+simple ones that can be detected.
+        </p>
+
+        <p>
+<a>Verifiers</a> SHOULD not request bearer claims that can be used to correlate
```

Comment:
Perhaps "unduly correlate" is better than just "correlate". It's about information minimization.

Reply:
This section would be more correct if it talked about bearer credentials rather than bearer claims. A claim that is not signed is not of much value.

Reply:
Done, both suggestions applied to text.